README.md

```diff
@@ -58,5 +58,39 @@ In a nutshell, a few EPICs:
 
 Keep in mind that the end goal is to use this package in any environment that can run a Go application, despite being a cluster or a local environment.
 
-As we evolve, evaluate the package with Camel-K team to make sure that it can fit their use case the same way it does today with their embedded package. 
+As we evolve, evaluate the package with Camel-K team to make sure that it can fit their use case the same way it does today with their embedded package.
 That might require some work to remove the relation of the integration concept from the build tasks.
+
+## Docker Registry configuration with Docker Cleaner
+
+With Docker to connect on a remote server must be set
+// Supported environment variables:
+- DOCKER_HOST to set the url to the docker server.
+- DOCKER_API_VERSION to set the version of the API to reach, leave empty for latest.
+- DOCKER_CERT_PATH to load the TLS certificates from.
+- DOCKER_TLS_VERIFY to enable or disable TLS verification, off by default.
+
+otherwise a local docker registry will be used if nothing is present
+
+## Docker Registry configuration with Podman Cleaner
+
+With podman to connect can be used
```

Comment:
same, maybe Parameters to connect with Podman.

Reply:
ok